### DISCUSSION TITLE
Samsung s-pen button access

### DISCUSSION THREAD

**Discussion by ctwhome** [tauri-apps/tauri#8415] [Q&A]
Hi 👋

I am trying to use tauri v2 to create a multiplatform drawing application using webgl, i wonder if eventually i could access the Samsung pen sdk to be able to interact with the spen button (for erasing, selecting.. Etc) or if at the contrary I would have to develop a native android module that somehow sends the spen events to the tauri app. 

Thanks so much already for your answers. ✏

**Comment by FabianLars** [comment]
The only way i see is to include the sdk in the android studio project attached to your tauri app, or a seperate tauri plugin (i would typically recommend the latter). https://beta.tauri.app/guides/plugins/develop-mobile/
I did something similar for a closed source project and it felt, well alright 🤷

**Reply by ctwhome** [reply]
So you couldn't get it to work?

**Reply by FabianLars** [reply]
The way I described it worked fine for me.

Edit: it was not the s pen sdk i integrated but looking at the docs it shouldn't be that different.

**Reply by ctwhome** [reply]
Thank you very, very much for the hints. I'm amazed that it is all possible with tauri, at some point I'll have to deal with it and the apple pencil ✏!
